gabrielaEnache/src/main/java/com/endava/io/NewIOUtils.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.io;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class NewIOUtils {
+    public static Path getFilePath(String pathToFile) {
+        return Paths.get(pathToFile);
+    }
+
+    public static int countWordInFile(String word, Path path) {
+        int result = 0;
+        try (BufferedReader reader = Files.newBufferedReader(path)) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                if (line.equalsIgnoreCase(word)) {
+                    result++;
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return result;
+    }
+
+    public static Path createFile(String path, String filename) {
+        Path directory = Paths.get(path);
+        return directory.resolve(filename);
+    }
+
+    public static void writeInFile(String message, Path path) {
```

Comment:
you could try to read and write using FileChannel

Reply:
Thank you for suggestion. I will have a look.